pyomo/common/tests/test_flags.py

```diff
@@ -41,3 +41,9 @@ def test_NOTSET(self):
             del sys.modules['sphinx']
             in_testing_environment(None)
         self.assertIsNone(in_testing_environment.state)
+
+    def test_singleton(self):
+        # This tests that the type is a "singleton", and attempts to
+        # construct an instance will return the class
```

Comment:
Grammar weirdness here - maybe it's supposed to be this?
```suggestion
        # construct an instance that will return the class
```